Runtime/AvatarCreator/Scripts/Data/CreatedUser.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
```

Comment:
Seems none of these using statements are required. They can be removed.
![image](https://github.com/user-attachments/assets/bbf4b95d-1a89-4fba-ad9a-cdc17f52ab3e)